src/WorkOS.net/Services/Webhooks/Entities/Webhook.cs

```diff
@@ -24,5 +24,11 @@ public class Webhook
         /// </summary>
         [JsonProperty("data")]
         public object Data { get; set; }
+
+        /// <summary>
+        /// The created_at timestamp of the event.
```

Comment:
Could we follow the style we use for our other `CreatedAt` timestamp fields?

https://github.com/workos/workos-dotnet/blob/9a129a697234fcc85d9ec2c8d4ba871dc2129e56/src/WorkOS.net/Services/DirectorySync/Entities/Directory.cs#L57-L61

```suggestion
        /// The timestamp of when the event was created.
```